sys/linux/dev_dma_heap.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <asm/ioctl.h>
+include <uapi/linux/fcntl.h>
+include <uapi/linux/dma-heap.h>
+
+resource fd_dma_heap[fd]
+
+openat$dma_heap(fd const[AT_FDCWD], file ptr[in, string["/dev/dma_heap"]], flags flags[open_flags], mode const[0]) fd_dma_heap
```

Comment:
TODO: recheck if it's actually `/dev/dma_heap` that we should use and whether it exists on our VMs.

Reply:
Confirmed that `/dev/dma_heap/system` exists on `buildroot`-derived VMs created by `tools/create-buildroot-image.sh`